README.md

```diff
@@ -105,6 +105,30 @@ logger = logging.getLogger(__name__)
 logger.info('testing123...')
 ```
 
+## Clients
+Client connects with server.py on host machine to communicate with modules. Here is an example of creating a client.
+```python
+from client import client
+client_1 = client()
```

Comment:
let's add `'localhost'` as the argument (even though it is default). On a separate question, is the argument name in client's `__init__` method called `host`. If not, can we change it to that? If it is, let's explicitly say here `host = 'localhost'`.

Reply:
Also superrrrrr nitpicky - can we change `client_1` to `myclient` or something similar?